content/en/docs/languages/java/configuration.md

```diff
@@ -0,0 +1,413 @@
+---
+title: Configuration
+linkTitle: Configuration
+weight: 10
+aliases: [config]
+# prettier-ignore
+cSpell:ignore: authservice blrp Dotel ignore LOWMEMORY myservice ottrace PKCS retryable
+---
+
+The OpenTelemetry SDK provides a working implementation of the API, and can be
+set up and configured in a number of ways. The Java SDK supports most of the
+available [configuration options](/docs/languages/sdk-configuration/). For
+conformance details, see the
+[compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md).
+
+{{% alert title="System Properties and Environment Variables" color="info" %}}
+Any setting configurable with a system property can also be configured with an
+environment variable. Apply the following steps to convert a system property to
+an environment variable:
+
+- Convert the name to uppercase.
+- Replace all `.` and `-` characters with `_`.
+
+For example `otel.instrumentation.common.default-enabled` would convert to
+`OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED`.
+
+{{% /alert %}}
+
+## General
+
+The autoconfigure module registers Java shutdown hooks to shut down the SDK when
```

Comment:
Let's link to or provide some context on what the autoconfigure module actually is. We have to assume the reader doesn't know what the autoconfigure module is when they land on this page. Maybe a link to this: http://localhost:1313/docs/languages/java/instrumentation/#automatic-configuration

And that section should probably link back to this page for a full list of all options.